workers/unity/Packages/com.improbable.gdk.core/Subscriptions/StandardSubscriptionManagers/LogDispatcherSubscriptionManager.cs

```diff
@@ -0,0 +1,37 @@
+using Improbable.Gdk.Core;
+using Unity.Entities;
+
+namespace Improbable.Gdk.Subscriptions
+{
+    [AutoRegisterSubscriptionManager]
+    public class LogDispatcherSubscriptionManager : SubscriptionManager<ILogDispatcher>
+    {
+        private Subscription<ILogDispatcher> subscription;
+        private readonly ILogDispatcher logger;
```

Comment:
Nitpick - style we are generally using is to put readonly over the mutable ones.